woocommerce/class-sv-wc-plugin-compatibility.php

```diff
@@ -322,9 +322,33 @@ public static function is_wc_subscriptions_version_lt( $version ) {
 	 *
 	 * @return string|null WooCommerce Subscriptions version number or null if not found
 	 */
-	protected static function get_wc_subscriptions_version() {
+	protected static function get_wc_subscriptions_version() : ?string {
 
-		return class_exists( 'WC_Subscriptions' ) && ! empty( \WC_Subscriptions::$version ) ? \WC_Subscriptions::$version : null;
+		$version = null;
+
+		if ( class_exists( 'WC_Subscriptions' ) && ! empty( \WC_Subscriptions::$version ) ) {
+			$version = \WC_Subscriptions::$version;
+		}
+
+		if ( ! $version && class_exists( 'WC_Subscriptions_Core_Plugin' ) ) {
```

Comment:
```suggestion
		if ( class_exists( 'WC_Subscriptions' ) && ! empty( \WC_Subscriptions::$version ) ) {
			$version = \WC_Subscriptions::$version;
		} else if ( ! $version && class_exists( 'WC_Subscriptions_Core_Plugin' ) ) {
```

Do you think this should be `else if` instead of completely override the version by the core package?

Reply:
sure, it would work either way